bench/bench.ml

```diff
@@ -0,0 +1,74 @@
+open Bechamel
+open Toolkit
+open Ocamlformat_lib
+
+type range = int * int
+
+type input =
+  { name: string
+  ; input_name: string
+  ; kind: Syntax.t
+  ; source: string
+  ; conf: Conf.t
+  ; action: [`Format | `Numeric of range] }
+
+let inputs =
+  let source_ml = Stdio.In_channel.read_all "test/source_bench.ml" in
```

Comment:
```suggestion
  let source_ml = Stdio.In_channel.read_all "test/source.ml" in
```

Reply:
The idea was to concatenate 5 times source.ml and make the result into source_bench.ml, dune should be able to generate source_bench.ml from the rules.

Reply:
Does it depend on a specific dune version? I can't seem to make it work even locally.